sql/src/test/java/io/crate/integrationtests/CustomSchemaIntegrationTest.java

```diff
@@ -92,6 +99,21 @@ public void testGetCustomSchema() throws Exception {
 
         execute("select id from custom.t where id in (2,4) order by id");
         assertThat(TestingHelpers.printedTable(response.rows()), is("2\n4\n"));
+    }
+
+    @Test
+    public void testSelectFromDroppedTableWithMoreThanOneTableInSchema() {
+        expectedException.expectMessage("Table 'custom.foo' unknown");
 
+        execute("create table custom.foo (id integer)");
+        execute("create table custom.bar (id integer)");
+        ensureYellow();
```

Comment:
I think the ensureYellow isn't necessary here since you don't rely on shards to be allocated